package.json

```diff
@@ -28,21 +28,25 @@
     "@mapbox/rehype-prism": "^0.3.0",
     "@types/fs-extra": "^5.0.4",
     "@types/jest": "^23.3.10",
+    "@types/klaw-sync": "^6.0.0",
     "@types/node": "~10.12.23",
     "@types/node-fetch": "^2.1.6",
     "canonical-path": "^1.0.0",
     "chokidar-cli": "^1.2.1",
     "codecov": "^3.0.4",
+    "feed": "^2.0.4",
     "fs-extra": "^7.0.0",
     "hast-to-hyperscript": "^5.0.0",
     "hast-util-to-string": "^1.0.1",
     "jest": "~23.4.1",
     "jest-junit": "~3.6.0",
+    "klaw-sync": "^6.0.0",
```

Comment:
Do we use this?

Reply:
My code was using this to recursively walk directories for blog entries. It looks like this code needs the entries to be listed explicitly.

Reply:
Will remove this. Since blog index's block is already getting the file list (and contents) i have just having it pass that data into the rss feed generator.